swatch-contracts/src/main/resources/application.properties

```diff
@@ -318,12 +318,14 @@ mp.messaging.incoming.offering-sync-umb.connector=smallrye-amqp
 mp.messaging.incoming.offering-sync-umb.address=${OFFERING_UMB_QUEUE}
 mp.messaging.incoming.offering-sync-umb.client-options-name=umb
 mp.messaging.incoming.offering-sync-umb.enabled=${UMB_ENABLED}
+mp.messaging.incoming.offering-sync-umb.failure-strategy=ignore
```

Comment:
Need to use a different value for these consumers since they use amqp; out of the [supported values](https://smallrye.io/smallrye-reactive-messaging/4.24.0/amqp/receiving-amqp-messages/#failure-management), I'm inclined to just use `accept` as it's the closest in behavior to `ignore`.